reana_db/models.py

```diff
@@ -286,22 +286,71 @@ class JobStatus(enum.Enum):
     queued = 5
 
 
+class UserWorkflowSession(Base, Timestamp):
+    """User Workflow Session table."""
+
+    __tablename__ = "user_workflow_session"
+    __table_args__ = {"schema": "__reana"}
+
+    user_id = Column(UUIDType, ForeignKey("__reana.user_.id_"), nullable=False)
+    workflow_id = Column(UUIDType, ForeignKey("__reana.workflow.id_"), nullable=True)
+    session_id = Column(
+        UUIDType, ForeignKey("__reana.interactive_session.id_"), primary_key=True
+    )
+
+    def __repr__(self):
+        """User Workflow Session string representation."""
+        return "<UserWorkflowSession {} {} {}>".format(
+            self.user_id, self.session_id, self.workflow_id
+        )
+
+
+class InteractiveSessionType(enum.Enum):
+    """Enumeration of interactive session types."""
+
+    jupyter = 0
+
+
+class InteractiveSession(Base, Timestamp):
```

Comment:
We are missing the `InteractiveSessionResource` table ([like `WorkflowResource`](https://github.com/reanahub/reana-db/blob/8203b6afeb90df9d39c13a4c00e71c43cd0e9077/reana_db/models.py#L667)), we will need it to account resource usage in InteractiveSessions (more [here](https://github.com/reanahub/reana-db/pull/92/files#r501686934)).


For now, we should measure just CPU usage, not disk usage since now the workspace is the same as the workflow and it is already accounted (which would change if we go for https://github.com/reanahub/reana-db/pull/90/files#r501548133). Can you remember to open an issue for this, "accounting for `InteractiveSessions` CPU usage" once we merge this?